import { BadRequestException } from '../../src/utils/customError.util';

describe('CustomError를 테스트한다', () => {});
